src/DataSource.ts

```diff
@@ -26,6 +26,7 @@ import {
 } from './types';
 import { transformResponse } from 'utils/transform';
 import { XRayLanguageProvider } from 'language_provider';
+import { TimeRange } from '@grafana/data/types/time';
```

Comment:
Can this import merged with line 10?

Reply:
Uh yeah weird autocomplete, ideally this should be eslinted or something cause it's definitely not a good idea to have imports going deep into the package, instead of importing from top level.